.env.example

```diff
@@ -70,6 +70,9 @@ OAUTH_KEYCLOAK_SECRET=
 OAUTH_KEYCLOAK_URI=
 OAUTH_KEYCLOAK_REALM=
 OAUTH_KEYCLOAK_VERSION=
+OAUTH_ZITADEL_ID=
```

Comment:
does Mbin keeps working without adding this to the config?

Reply:
It should, but since that file is supposed to show the possible keys I think its fine

Reply:
so what I mean, is most current admin / server owners will forget to add this configuration. So Mbin should still work fine, even if `OAUTH_ZITADEL_ID` is undefined (not present in the `.env` file).